libraries/plugin.rb

```diff
@@ -117,14 +117,14 @@ def whyrun_supported?
             new_resource.source,
             new_resource.name,
             nil,
-            cli_opts: new_resource.options,
+            cli_opts: new_resource.options
```

Comment:
rather than changing all of these I think we just want to update the .rubocop.yml